README.md

```diff
@@ -5,11 +5,15 @@ A simple load generator for the crapi application that uses Selenium to exercise
 
 ```bash
 ./gradlew buildDockerImage
-docker run -e CRAPI_URL=<crapi url> levoai/crapi-loadgen
+docker run -e CRAPI_URL=http://host.docker.internal:8090 levoai/crapi-loadgen
+
+# Run using a proxy
+docker run -e CRAPI_URL=http://host.docker.internal:8090 -e HTTP_PROXY=http://host.docker.internal:8080 levoai/crapi-loadgen
```

Comment:
I presume this will work as long as proxy(running on same host) opens port 8080 and listens even if it is running in a separate docker container.

Reply:
That's right.